src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -634,6 +634,35 @@ public static boolean equals(final Object object1, final Object object2) {
         return object1.equals(object2);
     }
 
+    /**
+     * <p>Compares given {@code object} to a vararg of {@code searchObjects}</p>
+     *
+     * <pre>
+     * ObjectUtils.equalsAny(null)                                      = false
+     * ObjectUtils.equalsAny(null, null, null)                          = true
+     * ObjectUtils.equalsAny("", "")                                    = true
+     * ObjectUtils.equalsAny(Boolean.TRUE, null)                        = false
+     * ObjectUtils.equalsAny(Boolean.TRUE, "true")                      = false
+     * ObjectUtils.equalsAny(Boolean.TRUE, Boolean.TRUE)                = true
+     * ObjectUtils.equalsAny(Boolean.TRUE, Boolean.FALSE, Boolean.TRUE) = true
+     * </pre>
+     *
+     * @param object to compare, may be {@code null}
+     * @param searchObjects a vararg of objects, may be {@code null}.
+     * @return {@code true} if {@code object} is equal to any of the {@code searchObjects}.
+     */
+    public static boolean equalsAny(final Object object, final Object... searchObjects) {
+        if (ArrayUtils.isEmpty(searchObjects)) {
```

Comment:
Or just `searchObjects== null ? false : Stream.of(searchObjects).anyMatch(e -> Objects.equals(object, e));`
-1 anyway since this duplicated the functionality in `ArrayUtils.contains(Object[], Object)`